<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>420</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>680</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DLSim</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>MAC Algorithm</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Number of devices</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="btQuit">
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Data transferences</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="btStart">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QListWidget" name="lwEvents"/>
    </item>
    <item row="0" column="3">
     <widget class="QComboBox" name="cbAlg">
      <item>
       <property name="text">
        <string>TDMA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CSMA/CA</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="btAbout">
      <property name="text">
       <string>About</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="sbDevices">
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btAddLoad">
        <property name="text">
         <string>Add event</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btUpdateSel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Update selected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btRemoveSel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Remove selected</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Random seed (-1 for non-deterministic behaviour)</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="sbSeed">
      <property name="minimum">
       <number>-1</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Bit Error Rate (BER)</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QDoubleSpinBox" name="sbBER">
      <property name="decimals">
       <number>6</number>
      </property>
      <property name="maximum">
       <double>0.999999000000000</double>
      </property>
      <property name="singleStep">
       <double>0.000001000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
